client/src/components/Input.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const InputWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+`;
+const Arrow = styled.i`
+  border: solid black;
+  border-width: 0 3px 3px 0;
+  margin-right: 15px;
+  display: inline-block;
+  padding: 3px;
+  transform: rotate(-45deg);
+  -webkit-transform: rotate(-45deg);
+`;
+
+const Input = styled.input`
+  margin-bottom: 50px;
+  border: none;
+  margin: 0;
+  border-bottom: 1px solid rgba(211, 211, 211, 1);
```

Comment:
You wanna work with RGB color definitition? Your choice of course

Reply:
Thx for your feedback. Using rgba enables the possibility to change the opacity on here.